modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
++
+.Example of enabling larger results using sidecar logs
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    options:
+      configMaps:
+        feature-flags:
+          data:
+            results-from: "sidecar-logs"
+----
+
+* Before this update, parameter propagation was allowed in `PipelineRun` and `TaskRun` resources, but not in the `Pipeline` resource. With this update, you can propagate `params` in the `Pipeline` resource down to the inlined pipeline `tasks` and its inlined `steps`. Wherever a resource, such as `PipelineTask` or `StepAction`, is referenced, you must pass the parameters explicitly. 
++
+.Example of using `params` within a pipeline
+[source,yaml]
+----
+apiVersion: tekton.dev/v1 # or tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pipeline-propagated-params
+spec:
+  params:
+    - name: HELLO
+      default: "Hello World!"
+    - name: BYE
+      default: "Bye World!"
+  tasks:
+    - name: echo-hello
+      taskSpec:
+        steps:
+          - name: echo
+            image: ubuntu
+            script: |
+              #!/usr/bin/env bash
+              echo "$(params.HELLO)"
+    - name: echo-bye
+      taskSpec:
+        steps:
+          - name: echo-action
+            ref:
+              name: step-action-echo
+            params:
+              - name: msg
+                value: "$(params.BYE)" 
+----
+
+* With this update, you can use the task run or pipeline run definition to configure the compute resources for steps and sidecars in a task. 
++
+.Example task for configuring resources
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: side-step
+spec:
+  steps:
+    - name: test
+      image: docker.io//alpine:latest
+  sidecars:
+    - name: side
+      image: docker.io/linuxcontainers/alpine:latest
+----
++
+.Example `TaskRun` definition that configures the resources
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: TaskRun
+metadata:
+  name: test-sidestep
+spec:
+  taskRef:
+    name: side-step
+  stepSpecs:
+    - name: test
+      computeResources:
+        requests:
+          memory: 1Gi
+  sidecarSpecs:
+    - name: side
+      computeResources:
+        requests:
+          cpu: 100m
+        limits:
+          cpu: 500m
+----
+
+[id="Operator-new-features-1-16_{context}"]
+=== Operator
+
+* With this update, {pipelines-shortname} includes the `git-clone` `StepAction` definition for a step that clones a Git repository. Use the HTTP resolver to reference this definition. The URL for the definition is `https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml`. The `StepAction` definition is also installed in the `openshift-pipelines` namespace, however, the cluster resolver does not support `StepAction` definitions at present.
```

Comment:
Escape character needed for the URL?  See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#example-urls

Reply:
None of the situations described in the example URLs sections apply here. We are not creating an example here that we would not want to render, nor want to create an external link. For this reason, I am leaving this as it is.